Automatic Validation ended with:
> 2024-08-12T18:10:17.1540431Z Manifest Validation Failed
 2024-08-12T18:10:22.2008234Z ##[error]Manifest Validation Failed

(Automated response - build 887.)

--- a/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.installer.yaml
@@ -1,0 +1,23 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.1.13
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSuccessCodes:
+- 1223
+UpgradeBehavior: install
+ReleaseDate: 2024-08-08
+AppsAndFeaturesEntries:
+- DisplayVersion: "2024.1"
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x64
+  InstallerUrl: https://redirector.gvt1.com/edgedl/android/studio/install/2024.1.1.13/android-studio-2024.1.1.13-windows.exe
+  InstallerSha256: D8F333687C6DE52F38AF40ED4CEBAF15E6BA82A53E5B07FD2161E3F2412FA5FE
+ManifestType: installer
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.locale.en-US.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.locale.en-US.yaml
@@ -1,0 +1,44 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.1.13
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/studio/intro
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android Studio
+PackageUrl: https://developer.android.com/studio
+License: Android Software Development Kit License Agreement
+LicenseUrl: https://developer.android.com/studio/terms
+Copyright: Copyright © 2000–2024 Google
+CopyrightUrl: https://developer.android.com/studio/terms
+ShortDescription: The official Integrated Development Environment (IDE) for Android app development.
+Description: |-
+  Android Studio is the official Integrated Development Environment (IDE) for Android app development.
+  Based on the powerful code editor and developer tools from IntelliJ IDEA,
+  Android Studio offers even more features that enhance your productivity when building Android apps.
+Moniker: android-studio
+Tags:
+- aab
+- adb
+- android
+- androidstudio
+- apk
+- app
+- debug
+- development
+- emulator
+- fastboot
+- ide
+- intellij
+- studio
+ReleaseNotesUrl: https://developer.android.com/studio/releases
+Documentations:
+- DocumentLabel: Troubleshoot
+  DocumentUrl: https://developer.android.com/studio/troubleshoot
+- DocumentLabel: Known Issues
+  DocumentUrl: https://developer.android.com/studio/known-issues
+ManifestType: defaultLocale
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.1.13/Google.AndroidStudio.yaml
@@ -1,0 +1,8 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.1.13
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.6.0

